### PR TITLE
Remove SWIG dependency from the project

### PR DESCRIPTION
**Pull request by neewy**
### Description of the Change

This pull request deletes code files, build files, descriptions, CI files, and dependencies related to SWIG-generated libraries 

### Benefits

No legacy anymore

### Possible Drawbacks 

None

### Usage Examples or Tests *[optional]*

Build Iroha and check if it builds (and passes CI)

### CODE REVIEW

File: shared_model/backend/protobuf/queries/proto_blocks_query.hpp [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 namespace shared_model {
   namespace proto {
-    class BlocksQuery FINAL : public CopyableProto<interface::BlocksQuery,
+    class BlocksQuery : public CopyableProto<interface::BlocksQuery,
```
Comment (luckychess) [inline shared_model/backend/protobuf/queries/proto_blocks_query.hpp]: Use `final` instead (here and in every other place with FINAL)

File: shared_model/backend/protobuf/queries/proto_query.hpp [hunk 2]
```diff
@@ -13,7 +13,7 @@
 namespace shared_model {
   namespace proto {
 
-    class Query FINAL : public interface::Query {
+    class Query : public interface::Query {
```
Comment (luckychess) [inline shared_model/backend/protobuf/queries/proto_query.hpp]: same

File: shared_model/backend/protobuf/queries/proto_query_payload_meta.hpp [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
 namespace shared_model {
   namespace proto {
-    class QueryPayloadMeta FINAL
+    class QueryPayloadMeta
```
Comment (luckychess) [inline shared_model/backend/protobuf/queries/proto_query_payload_meta.hpp]: same

File: shared_model/backend/protobuf/transaction.hpp [hunk 4]
```diff
@@ -11,8 +11,7 @@
 
 namespace shared_model {
   namespace proto {
-
-    class Transaction FINAL : public interface::Transaction {
+      class Transaction : public interface::Transaction {
```
Comment (luckychess) [inline shared_model/backend/protobuf/transaction.hpp]: same

File: shared_model/builders/protobuf/builder_templates/blocks_query_template.hpp [hunk 5]
```diff
@@ -28,7 +28,7 @@ namespace shared_model {
     template <int S = 0,
               typename SV = validation::DefaultUnsignedBlocksQueryValidator,
               typename BT = UnsignedWrapper<BlocksQuery>>
-    class DEPRECATED TemplateBlocksQueryBuilder {
+    class TemplateBlocksQueryBuilder {
```
Comment (luckychess) [inline shared_model/builders/protobuf/builder_templates/blocks_query_template.hpp]: Replace to `[[deprecated]]`.

File: shared_model/builders/protobuf/builder_templates/query_template.hpp [hunk 6]
```diff
@@ -30,7 +30,7 @@ namespace shared_model {
     template <int S = 0,
               typename SV = validation::DefaultUnsignedQueryValidator,
               typename BT = UnsignedWrapper<Query>>
-    class DEPRECATED TemplateQueryBuilder {
+    class TemplateQueryBuilder {
```
Comment (luckychess) [inline shared_model/builders/protobuf/builder_templates/query_template.hpp]: same

File: shared_model/builders/protobuf/builder_templates/transaction_template.hpp [hunk 7]
```diff
@@ -33,7 +33,7 @@ namespace shared_model {
     template <int S = 0,
               typename SV = validation::DefaultUnsignedTransactionValidator,
               typename BT = UnsignedWrapper<Transaction>>
-    class DEPRECATED TemplateTransactionBuilder {
+    class TemplateTransactionBuilder {
```
Comment (luckychess) [inline shared_model/builders/protobuf/builder_templates/transaction_template.hpp]: same

File: shared_model/builders/protobuf/transaction_responses/proto_transaction_status_builder.hpp [hunk 8]
```diff
@@ -10,7 +10,7 @@
 
 namespace shared_model {
   namespace proto {
-    class DEPRECATED TransactionStatusBuilder {
+    class TransactionStatusBuilder {
```
Comment (luckychess) [inline shared_model/builders/protobuf/transaction_responses/proto_transaction_status_builder.hpp]: same

File: shared_model/builders/protobuf/transaction_sequence_builder.hpp [hunk 9]
```diff
@@ -18,7 +18,7 @@ namespace shared_model {
      * @tparam SV Stateless validator type
      */
     template <typename SV>
-    class DEPRECATED TransportBuilder<interface::TransactionSequence, SV> {
+    class TransportBuilder<interface::TransactionSequence, SV> {
```
Comment (luckychess) [inline shared_model/builders/protobuf/transaction_sequence_builder.hpp]: same

File: shared_model/builders/protobuf/transport_builder.hpp [hunk 10]
```diff
@@ -18,7 +18,7 @@ namespace shared_model {
      * @tparam SV Stateless validator type
      */
     template <typename T, typename SV>
-    class DEPRECATED TransportBuilder {
+    class TransportBuilder {
```
Comment (luckychess) [inline shared_model/builders/protobuf/transport_builder.hpp]: same

File: shared_model/builders/protobuf/unsigned_proto.hpp [hunk 11]
```diff
@@ -22,7 +22,7 @@ namespace shared_model {
      * finish() throws an exception
      */
     template <typename T>
-    class DEPRECATED UnsignedWrapper {
+    class UnsignedWrapper {
```
Comment (luckychess) [inline shared_model/builders/protobuf/unsigned_proto.hpp]: same

File: shared_model/builders/transaction_responses/transaction_status_builder.hpp [hunk 12]
```diff
@@ -19,7 +19,7 @@ namespace shared_model {
      * @tparam BuilderImpl
      */
     template <typename BuilderImpl>
-    class DEPRECATED TransactionStatusBuilder {
+    class TransactionStatusBuilder {
```
Comment (luckychess) [inline shared_model/builders/transaction_responses/transaction_status_builder.hpp]: same

File: test/module/shared_model/builders/protobuf/builder_templates/block_template.hpp [hunk 13]
```diff
@@ -30,7 +30,7 @@ namespace shared_model {
     template <int S = 0,
               typename SV = validation::DefaultUnsignedBlockValidator,
               typename BT = UnsignedWrapper<Block>>
-    class DEPRECATED TemplateBlockBuilder {
+    class TemplateBlockBuilder {
```
Comment (luckychess) [inline test/module/shared_model/builders/protobuf/builder_templates/block_template.hpp]: Again `[[deprecated]]`

File: test/module/shared_model/builders/protobuf/builder_templates/proposal_template.hpp [hunk 14]
```diff
@@ -22,7 +22,7 @@ namespace shared_model {
      * @tparam SV -- stateless validator called when build method is invoked
      */
     template <int S = 0, typename SV = validation::DefaultProposalValidator>
-    class DEPRECATED TemplateProposalBuilder {
+    class TemplateProposalBuilder {
```
Comment (luckychess) [inline test/module/shared_model/builders/protobuf/builder_templates/proposal_template.hpp]: same

File: docs/source/guides/libraries/python.rst [hunk 15]
```diff
@@ -1,309 +1,42 @@
-.. TODO: IR-1847 nickaleks 05.11.18 rework documentation with native python lib
 Python Library
 --------------
 
 Where to Get
 ^^^^^^^^^^^^
 
-There are two ways to get Iroha library for Python: via pip and manual compilation from source code. The installation via pip do the same steps as manual compilation so for both of them you need to install all of the prerequisites.
+A supported python library for Iroha is available at its `own Hyperledger iroha-python repo <https://github.com/hyperledger/iroha-python/>`__.
+Python 3.6+ is supported.
```
[the remark this answers is not in the document]
Comment (neewy) [reply docs/source/guides/libraries/python.rst]: Yes

[the remark this answers is not in the document]
Comment (luckychess) [reply docs/source/guides/libraries/python.rst]: Oh, really?

[the remark this answers is not in the document]
Comment (l4l) [reply docs/source/guides/libraries/python.rst]: Note that at ubuntu 16 it's only python3.5 at repo

File: shared_model/builders/protobuf/builder_templates/transaction_template.hpp [hunk 16]
```diff
@@ -33,7 +33,7 @@ namespace shared_model {
     template <int S = 0,
               typename SV = validation::DefaultUnsignedTransactionValidator,
               typename BT = UnsignedWrapper<Transaction>>
-    class TemplateTransactionBuilder {
+    class TemplateTransactionBuilder TemplateTransactionBuilder {
```
Comment (luckychess) [inline shared_model/builders/protobuf/builder_templates/transaction_template.hpp]: I guess you mean [[deprecated]] here.

File: test/integration/binary/CMakeLists.txt [hunk 17]
```diff
@@ -47,3 +47,5 @@ add_test(
     "ROOT_DIR=${PROJECT_SOURCE_DIR}"
     $<TARGET_FILE:binary_test> --gtest_filter=*/0*
 )
+
+add_subdirectory(bindings)
```
Comment (igor-egorov) [inline test/integration/binary/CMakeLists.txt]: Need to add a trailing newline

File: test/integration/binary/bindings/model_crypto.cpp [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+#include "model_crypto.hpp"
+#include "common/byteutils.hpp"
+#include "cryptography/crypto_provider/crypto_defaults.hpp"
+#include "generator/generator.hpp"
+
+namespace shared_model {
+  namespace bindings {
+    crypto::Keypair ModelCrypto::generateKeypair() {
+      return crypto::CryptoProviderEd25519Sha3::generateKeypair();
+    }
```
Comment (igor-egorov) [inline test/integration/binary/bindings/model_crypto.cpp]: Don't merge this. This commit cannot fix BTF

**Pull request merged**